### ISSUE TITLE
[Feature] [Question]: Does 'globe' Support Transparent baseColor?

### ISSUE THREAD

**Issue by cenkakay** [apache/echarts#19989]
### What problem does this feature solve?

Is there a way to see what's behind the sphere? Like the example in the link below.
https://www.robscanlon.com/encom-globe/

```
// This works
 globe: {
            shading: 'color',
            baseColor: 'red',
}

// This not works!
 globe: {
            shading: 'color',
            baseColor: 'transparent',
}
```

### What does the proposed API look like?

```
 globe: {
            shading: 'color',
            baseColor: 'transparent',
}
```

**Comment by helgasoft** [comment]
> Does 'globe' Support Transparent baseColor?

yes, but probably not how you want it. 
It seems there is no way of "painting" the globe with semi-transparent color or texture. Even when _detailTexture_ is a PNG with some transparency, the globe turns out opaque. One could change color opacity thru _baseColor_, but the globe remains a non-translucent 3D object.  [Demo](https://echarts.apache.org/examples/en/editor.html?gl=1&code=PYBwLglsB2AEC8sDeAoWsDmAbYAjApgFzJrqwDOAFgIYAmE0GxA5AMbA4BOzANKegHoBuauXwBhDsE6FmYTtWjkQ1TvmhhesIbADu0gNbl-24aIlSZzAMQAzew4AijrYIGxQ1VhDABPE-xcALLUYPicENRYxKhkZLT4YNQQWAAq-AAeYACuaiyUYGAg5IRC-Kw0nGDkAHTUKhX4NdIYAuoCEAC21Bj45AI4GMA1IIy8JuhhWbn4qSkMTLAAjAAMJgC-fHEAbhD4upIanBwxsNTZYMAASsBJYcS2UWI8Z1ggNMQAHCuw66SbpEuHEgIBiAPQYgifWIAG0TLE4rA_CAiLBmORWKEwpwAMwuLaI9jSejQUL4ADKvnIYU6LGweHw40RFF8nVwHHJEAAXqiYQAWHEvACsKwAugS4j58J1yX4sKiEczAtIWLgsNlGRLEZ5vH5iEstXF2ZwEpwAOoQWhgSj6w1kY2myRcFjWcRCgBiOIA7AA2ZgTX522ihaiwgPoJCwUmdVFyPqaF7bKIa2EAWiWqxeAE4Vi8lqLA-HkFHqDGWLRaFok-rebnYHXoNksFgC39maKNqQO-sANxAA)

**Comment by cenkakay** [comment]
> > Does 'globe' Support Transparent baseColor?
> 
> yes, but probably not how you want it. It seems there is no way of "painting" the globe with semi-transparent color or texture. Even when _detailTexture_ is a PNG with some transparency, the globe turns out opaque. One could change color opacity thru _baseColor_, but the globe remains a non-translucent 3D object. [Demo](https://echarts.apache.org/examples/en/editor.html?gl=1&code=PYBwLglsB2AEC8sDeAoWsDmAbYAjApgFzJrqwDOAFgIYAmE0GxA5AMbA4BOzANKegHoBuauXwBhDsE6FmYTtWjkQ1TvmhhesIbADu0gNbl-24aIlSZzAMQAzew4CCjrYIGxQ1VhDABPE-xcALLUYPicENRYxKhkZLT4YNQQWAAq-AAeYACuaiyUYGAg5IRC-Kw0nGDkAHTUKhX4NdIYAuoCEAC21Bj45AI4GMA1IIy8JuhhWbn4qSkMTLAAjAAMJgC-fHEAbhD4upIanBwxsNTZYMAASsBJYcS2UWI8Z1ggNMQAtABMAKyw61Im1Ilw4kBAMWB6DEET6xAA2iZYnFYH4QERYMxyKxQmFOABmAAi4xRsHY0no0FC-AAyr5yGFOixsHh8CSUeRfJ1cBwaRAAF4Y-EAFnxL1-KwAulsUT58J0aX4sBjkaTAtIWLgsNk2TLSZ5vH5iEs9SieZwEpwAOoQWhgSjG01xc2WyRcFjWcS_ABi-IA7AA2ZgTAFO2C0ULUBEh9BIWBUzoYuR9TQvbZRHUIz5LVYvACcKxeS0loZjyHj1ETLFotC06e1QsLsCb0GyWCwJcBpMlG1IPfWAG4gA)

Thank you for your answer. I thought maybe it was a special property. BTW, in the "scatter3D" type tooltip is not activated halfway through the item. I will open an issue about this.

**Comment by helgasoft** [comment]
> the "scatter3D" type tooltip is not activated halfway through the item.

yes, the click target is really small and bigger _symbolSize_ makes it worse.
Please note that ECharts GL/3D has its [own repo](https://github.com/ecomfe/echarts-gl/issues) for issue posting.
